package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
Please remove ^ as it is best practice to use fixed versions.

Reply:
Done @abhicool957